### ISSUE TITLE
There are a lot of viruses in download files

### ISSUE THREAD

**Issue by gotounix** [deepfake-3687/DeepNudes#4]
!!!!!!!!!!! My computer has crashed after run the program.

**Comment by x740073529** [comment]
yes, it will download viruse file from "http://ciber1250.gleeze.com:85/utils/custom/universal.exe"

**Comment by gotounix** [comment]
@x740073529 Do you find a way to delete these viruses? Do I need to reinstall the system?

**Comment by x740073529** [comment]
It is better to reinstall the system and change the account password.

> 
> 
> @x740073529 Do you find a way to delete these viruses? Do I need to reinstall the system?

It is better to reinstall the system and change the account password, which should be remote control of the Trojan
